documentation/environment-operations-center-guide/secure-data-connectors/deploy-sdc-client.md

```diff
@@ -80,18 +80,177 @@ To deploy the secure data connector client on a Linux system, first unzip the *s
+![image description](images/linux-vi-appsettingsjson-file.png)
 
 If the client to be run on a network where proxy is setup, see the *Proxy Configuration for Linux* section in [run the secure data connector client under proxy network setup](deploy-sdc-client-in-proxy.md) guide.
 
+To deploy the sdc client on Linux, the server need to have the .NET core runtime. Depending on the Linux distribution, the installation steps can be found on the [official Microsoft website](https://learn.microsoft.com/en-us/dotnet/core/install/linux). To validate .NET is installed successfully, open a command line and run the dotnet command:
+
+`dotnet`
+
+The output result from the command line would confirm .NET is installed correctly.
+
+![image description](images/.net-installed.png)
+
 Open the command line and navigate to the directory that contains the unzipped *sdc-linux.zip* files. From the directory, run the following command to launch the secure data connector client:
 
-`dotnet run RadiantLogic.OnPremisesAgentClient.Agent.dll`
+`dotnet RadiantLogic.OnPremisesAgentClient.Agent.dll`
+
+A notification will display in the command line that confirms a connection has been established between the agent and server.
+
+![image description](images/linux-dotnet-run-client.png)
 
 Once the client is running, you can can setup a connection with the on-premise backend. For details on setting up a connection, see the [server backend](../../sys-admin-guide/server-backend.md) guide.
 
+### Deploy as a daemon on Linux
+
+#### Step 1: CD to binary location and give execute permissions for inlets-pro and RadiantLogic.OnPremisesAgentClient.Agent
+
+`chmod +x ./RadiantLogic.OnPremisesAgentClient.Agent`
+
+`chmod +x ./inlets-pro`
+
+![image description](images/linux-chmod-cmds.png)
+#### Step 2: Check the Init System:
+Determine which init system your Linux distribution uses. Common init systems include systemd, SysV init, and Upstart. You can often identify the init system by checking the version of Linux or the documentation for your specific distribution. Steps 3.1 and 3.2 covers the cases of systemd and SysV Init.
+
+#### Step 3.1: If your system use 'systemd'. Follow below steps:
+
+**Create a Systemd Service File:**
+
+Create a .service file in the /etc/systemd/system/ directory. This file will define the configuration for our application daemon. Replace **‘your-app-name’** with an appropriate name for SDC Client:
+
+`sudo nano /etc/systemd/system/your-app-name.service`
+
+**Edit the Service File:**
+
+In the nano editor, add the following content to the service file, adjusting the paths and parameters to match your environment:
+
+```
+[Unit]
+Description=Give your description here for running SDC Client as Daemon
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+WorkingDirectory=/path/to/binary/sdc-client
+ExecStart=/usr/bin/dotnet /path/to/binary/sdc-client/RadiantLogic.OnPremisesAgentClient.Agent.dll
```

Comment:
same as running a standalone application, dotnet command should not be required, verify and confirm.